README.md

```diff
@@ -29,9 +29,9 @@ Use our Nixpkgs branch, add bootspec-secureboot as an input, and add our module
 ```nix
 # flake.nix
 {
-  inputs.nixpkgs.url = "github:DeterminateSystems/nixpkgs/boot-spec-unstable";
+  inputs.nixpkgs.url = "github:DeterminateSystems/nixpkgs/bootspec-rfc";
   inputs.bootspec-secureboot = {
-    url = "github:DeterminateSystems/bootspec-secureboot/update-to-rfc";
+    url = "github:DeterminateSystems/bootspec-secureboot/main";
```

Comment:
```suggestion
    url = "github:DeterminateSystems/bootspec-secureboot";
```

Would this not be the same? Or does nix flakes hard default to master?

Reply:
I don't think it was always the case that flakes got the proper default branch, so I'm inclined to keep it in case people are using older versions of Nix.